src/pages/docs/getting-started/basic-transfer.md

```diff
@@ -60,7 +60,11 @@ ckb-cli account export --lock-arg <lock_arg> --extended-privkey-path ./ckb/expor
 head -n 1 ./ckb/exported-key > ./ckb/key
 ```
 
-You can get Testnet funds from faucets: [https://faucet.nervos.org](https://faucet.nervos.org/).
+You can get Testnet funds from Faucets. Node1 requires CKB and RUSD, Node2 only requires CKB.
```

Comment:
why RUSD needed here? basic-transfer only involves transfer CKB right?

Reply:
Yes, RUSD is indeed needed here. Also mentioned in [fiber docs](https://github.com/nervosnetwork/fiber/blob/develop/docs/testnet-nodes.md#establishing-a-udt-channel-with-public-node-1).
This has also been confirmed with the developers.

Reply:
This [transaction](https://testnet.explorer.nervos.org/transaction/0x5000b7c922189169f64ecb291dbd941eb37e1befb1646d81f334967be2249dc3) is generated when open_channel is used.
However, other UDTs can also be used here (even self-issued), but RUSD is more convenient